web/pwa/src/assets/doc/cards.md

```diff
@@ -1 +1,44 @@
-On the "Cards" tab you can manage your virtual storage cards.
+# Virtual Card Manager
+
+**The 'Cards' tab allows you to manage your virtual storage cards.**
+
+
+## Background
+ Many Palm OS devices featured SD card expansion ports. Adding an SD card could be used for many purposes, such as storing addition programs, or storing large files or databases or multimedia. Some software was even distributed on SD cards.
+
+CloudPilot allows you to connect virtual storage cards to your emulated device.
+
+## Managing Cards
+### Adding a card
+ Tap the plus button in the top right corner to add a card. You can either create a blank card, or you can import an existing image file into CloudPilot.
+ 
+ **Importing a card**: Imported cards should be provided in a binary .img file format. 
```

Comment:
To be precise, the suffix is irrelevant, but the file should be an actual filesystem image. Most images can be used as SD cards, but only very specific sizes are suitable for use as memory sticks.